Expose data about past and current bill data ingestions

zanchi-r opened the pull request.

alexis-ld commented:
@hug33k : Overall it's good :) 

Just a couple changes : 

- Please remove the bow shadow on this panel. It's not good looking to use a box shadow to highlight a panel which is already on a panel itself. 
![screen shot 2017-11-08 at 16 11 06](https://user-images.githubusercontent.com/3480526/32556418-a7d91da0-c49f-11e7-9688-2682cd2244bd.png)
- On the same panel the tooltip on 'Reserved' is outside of the view with the new layout. Please make it appear on the bottom or on the right so we can see it.
![screen shot 2017-11-08 at 16 09 51](https://user-images.githubusercontent.com/3480526/32556459-c178bc3e-c49f-11e7-81c9-7d3736ad0b79.png)
- Clicking on 'Reserved' (still on the same panel) should display a list of reserved instances. It was working when this panel was on the home view. Please fix it :) .
- Change the button 'Show utilization' to 'Show metrics' or 'Show system metric'

hug33k commented:
- [x] Box shadow for `EC2 Instances`
- [x] Tooltip on `Reserved`
- [ ] List of reserved instances
- [x] Rename `Show utilization` to `Show metrics`

schubev committed:
server: register ingestion jobs in SQL database

schubev committed:
aws: expose info about current and past ingestions on the API

zanchi-r committed:
db: fixed aws_bill_update_job table name

zanchi-r committed:
main: register update completion in case of error

zanchi-r committed:
aws: fixed SQL request in BillRepositoryUpdates function

fixed SQL request in BillRepositoryUpdates function to always return the newest aws_bill_update_job

thibautcornolti commented on aws/routeAwsGet.go:
```diff
@@ -29,16 +31,148 @@ import (
+func BillRepositoryUpdates(db dbAccessor, userId int) ([]BillRepositoryUpdateInfo, error) {
+	// for each seleced bill repository, find data about the last and
+	// next/current update and join it all
+	var sqlstr = `
```
Not sure about the maintainability of that request
